### ISSUE TITLE
Error NLR-Parser DEBUG: Temporary directory

### ISSUE THREAD

**Issue by AG-Run** [steuernb/NLR-Annotator#11]
Hello

Im running NLR parser
java -jar NLR-Parser3.jar -t 7 -y /home/adriana/anaconda2/envs/meme1/bin/mast -x /home/adriana/Documentos/meme.xml -i RgenesCDS.fasta -c output.nlr.xml
I got a debug, that stacks and never generate a file, is it that normal?

DEBUG: Temporary directory is /home/adriana/Documentos/temp_mast1

Im installed MEME suite through conda environment, and Im using the latest version of ubuntu 20.04. The Mast version is 1.14.0 through bioconductor-mast.

Thanks for the help!

**Comment by steuernb** [comment]
Hi,
please try to run MAST on its own with a known NLR protein sequence (e.g. AGQ17382.1) and the provided meme.xml. This is just to exclude issues with the MEME installation or a corrupt meme.xml.
Thanks
Burkhard

**Comment by AG-Run** [comment]
Dear Steuernb

Thanks for the information. I realized all the problems can be solved if meme is installed correctly. In order to avoid the manual installation of all dependencies of Meme-Suite, and to make tests to find errors, it can be done in one step through a conda environment.

First I installed miniconda3, then I created and installed MEME-Suite in this environment using the command

_`conda create -n mem_env -c bioconda meme`_

Once meme is correctly installed, I activated the environment with

`conda activate mem_env`

Then, I run all the pipeline in your GitHub and it works perfectly

Thanks
